_data/projects/coala-analyzer.yml

```diff
@@ -0,0 +1,17 @@
+name: coala
+desc: coala provides a common multiplatform interface for linting and fixing all your code, regardless of the programming language you use. Its convenient user interface eliminates a lot of common tasks for the developer. coala also facilitates people entering the world of open source by providing them valuable feedback on coding standards and formatting. So, in a nutshell coala is a useful abstraction for static code analysis that is applicable to any text based language.
```

Comment:
This is probably a bit long, but I'm going to merge it in anyway - feel free to tweak this once the site is published in a few minutes.